packages/lesmis/src/utils/search.js

```diff
@@ -30,13 +33,17 @@ export const getPlaceIcon = type => {
 export const getOptionText = (option, entities, hierarchy = []) => {
   const { type, name, parentCode } = option;
 
-  const newHierarchy = [...hierarchy, name];
-
   if (type === 'country') {
-    return newHierarchy.join(', ');
```

Comment:
Refactor to only show the country text for the actual country result. In other words don't show it as part of the place name result on every result as it is too verbose and Country Name is always the same. eg. District Name, Province Name instead of District Name, Province Name, Country Name